disnake/abc.py

```diff
@@ -132,6 +132,19 @@ class User(Snowflake, Protocol):
         The user's username.
     discriminator: :class:`str`
         The user's discriminator.
+
+        .. note::
+            This is being phased out by Discord; the username system is moving away from ``username#discriminator``
+            to users having a globally unique username.
+            The value of a single zero (``"0"``) indicates that the user has been migrated to the new system.
+            See the `help article <https://dis.gd/app-usernames>`__ for details.
+    global_name: Optional[:class:`str`]
+        The user's global display name, if set.
+        This takes precedence over :attr:`.name` when shown.
+
+        For bots, this is the application name.
+
+        .. versionadded:: 2.9
```

Comment:
```suggestion
        .. versionadded:: 2.9

```
(as well as the other places)

Reply:
https://github.com/DisnakeDev/disnake/pull/1025/commits/de1d328f3cf92d9c51af73a81ea804272acab754